PYME/cluster/rulenodeserver.py

```diff
@@ -35,7 +35,7 @@ def template_fill(template, **kwargs):
     s = template
 
     for key, value in kwargs.items():
-        s = s.replace('{{%s}}' % key, '%s' % value)
+        s = s.replace('{{%s}}' % key, str(value))
```

Comment:
probably doesn't matter either way, but curious as to whether this change was needed. Did the original trigger an error?

Reply:
not needed, but after we ran into a weird issues with #403 /  #407 I thought it was worthwhile while we were here